README.md

```diff
@@ -103,10 +105,11 @@ Please file bugs as you see them! See [CONTRIBUTING.md](CONTRIBUTING.md) for mor
         version.
   - [ ] Adds additional imports when necessary.
   - [ ] Warns on other major breaking changes to elements.
-  - [ ] Upgrades mixins from 0.5 into behaviors of 1.0.
+  - [x] Upgrades mixins from 0.5 into behaviors of 1.0.
```

Comment:
It's hopefully a bit clearer if you expand the next few lines. This is part of upgrading official elements. So upgrading references of say Polymer.CoreResizable to Polymer.IronResizableBehavior in a mixin context.